README.md

```diff
@@ -129,67 +132,67 @@ public class DefaultController : Controller
 }
 ```
 
-Если нужно внутри Razor-шаблона вставить БЭМ-блок, используйте хелпер `@Html.Bem`, передав ему нужный bemjson.
+If you need to render BEM-block inside the Razor template you should use the `@Html.Bem` helper (and provide the essential bemjson for it).
 ```cs
 @Html.Bem(new { block = "my-block", data = Model })
 ```
 
-В разделе `bemSettings` файла Web.config вы можете настраивать, каким способом будут выбираться бандлы с bemhtml шаблонами для http-запросов. Возможны 3 варианта мэппинга запросов на бандлы:
+In the section named `bemSettings` of Web.config file you may setup how to selected the template-bundles for each http-request. 3 variants of request-to-bundle mapping are possible:
```

Comment:
In the section of Web.config file named `bemSettings` you can map http-requests with template bundles. There are 3 options of mapping available: